src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -69,6 +69,7 @@
 
             return total;
         }, {});
+        const udwConfigMove = JSON.parse(container.dataset.udwConfigMove);
```

Comment:
`udwConfigMove` means nothing to me. It's lacking the last part, in my opinion. Is it move to trash or move location?

Reply:
Is it for bulk move?

Reply:
Yes

Reply:
Then  maybe `udwConfigBulkMoveItems`

Reply:
Works for me. :)